docs/src/installation.md

```diff
@@ -41,6 +41,7 @@ is often more complex. We list below the currently available solvers.
 | [GLPK](http://www.gnu.org/software/glpk/)                                      | [GLPK.jl](https://github.com/JuliaOpt/GLPK.jl)                                   | GPL     | LP, MILP                           |
 | [Gurobi](http://gurobi.com)                                                    | [Gurobi.jl](https://github.com/JuliaOpt/Gurobi.jl)                               | Comm.   | LP, MILP, SOCP, MISOCP             |
 | [Ipopt](https://projects.coin-or.org/Ipopt)                                    | [Ipopt.jl](https://github.com/JuliaOpt/Ipopt.jl)                                 | EPL     | LP, QP, NLP                        |
+| [Juniper](https://github.com/lanl-ansi/Juniper.jl)                                    | [Juniper.jl](https://github.com/lanl-ansi/Juniper.jl)                                 | MIT     | MISOCP, MINLP                      |
```

Comment:
The second `|` is not aligned with other ones

Reply:
In my text editor there is no consistency in the alignment of `|` in the text.  Can you fix it as you prefer after the merge?